src/protocols/secure_channel/MessageCounterManager.h

```diff
@@ -101,13 +113,12 @@ class MessageCounterSyncMgr : public Messaging::ExchangeDelegate
      */
     struct RetransTableEntry
     {
-        RetransTableEntry() : protocolId(Protocols::NotSpecified) {}
-        ExchangeContext * exchangeContext; /**< The ExchangeContext for the stored CHIP message.
-                                                Non-null if and only if this entry is in use. */
+        RetransTableEntry() : peerNodeId(kUndefinedNodeId) {}
+
+        NodeId peerNodeId;                 /**< The peerNodeId of the message. kUndefinedNodeId if is not in use. */
+        SecureSessionHandle session;       /**< The session to send the message. */
+        PayloadHeader payloadHeader;       /**< The payload header for the message. */
```

Comment:
What is the size of `RetransTableEntry` before these changes?  After these changes?  How much memory does the array of these entries take?

Maybe it doesn't matter too much, since I think we should remove this entire retrans table in a followup...

Reply:
Yes, this will be removed